sources/packages/backend/workflow/test/models/assessment.model.ts

```diff
@@ -76,6 +76,7 @@ export interface AssessmentConsolidatedData extends JSONDoc {
   appealsStudentIncomeAppealData?: JSONDoc;
   appealsPartnerIncomeAppealData?: JSONDoc;
   appealsStudentDisabilityAppealData?: JSONDoc;
+  appealsStudentFinancialInformationAppealData?: JSONDoc;
```

Comment:
`appealsStudentFinancialInformationAppealData` can be typed in the same way that we also have `StudentDependent` type defined here.